.github/workflows/gating.yaml

```diff
@@ -0,0 +1,97 @@
+name: Gating
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install required packages
+      run: |
+        sudo apt-get install -y \
+          libkrb5-dev
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox tox-gh-actions
+    - name: Test with tox
+      run: tox
+    - name: Run coveralls-python
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        COVERALLS_FLAG_NAME: python-${{ matrix.python-version }}
+        COVERALLS_PARALLEL: true
+      run: |
+        pip3 install --upgrade pip
+        pip3 install --upgrade setuptools
+        pip3 install --upgrade coveralls
+        coveralls --service=github
+
+  coveralls-finish:
```

Comment:
We set up the coverage checking with the `.coveragerc` file. Is coveralls better? 

Could you please add a comment about a coverage percentage it uses here Maybe we even will not need the `coveragerc` file anymore ..

Reply:
coveralls provide results directly into PR and it has nice web interface for checking coverage diffs. Coveralls can measure overall decrease/increase in code coverage.

I'll check coveragerc, but in general coveragerc and coveralls are not mutually exclusive

Reply:
I'd keep  `coveragerc` as is